cypress/integration/Admin.feature

```diff
@@ -27,48 +27,87 @@ Feature: Admin
 
-  Scenario: Set and remove a system message on user facing site consecutively
+  @focus
+  Scenario: Remove system message 
     Given I visit the admin sign-in page
     And I log in to admin
     And I click "system-message-link"
-    Then I am taken to the system message page
-
-    # set message to display on front end
-    When I type "Your pizza is burning" into "message"
-    And I click "submit-button"
-    And I see "System message set" in the page text
-
     # remove message to display on front end
-    Then I clear the value in "message"
+    When I clear the value in "message"
     And I click "submit-button"
-    And I see "System message removed" in the page text
+    Then I see "System message removed" in the page text
+
+  @focus
+    # due to home page being on different url from admin, this scenario has to be seperate, but relies on previous scenario having run
+  Scenario: System message should now be removed on user-facing site. 
+    When I visit "/home"
+    Then I do not see "Your pizza is burning" in the page text
+    When I visit "/login"
+    Then I do not see "Your pizza is burning" in the page text
+    When I visit "/signup"
+    Then I do not see "Your pizza is burning" in the page text
 
+  @focus
+  Scenario: Set and remove a system message consecutively i:e having previously set and removed it, we do this again to ensure it still works
+    Given I visit the admin sign-in page
+    And I log in to admin
+    And I click "system-message-link"
     # set message to display on front end
     When I type "Your pizza is burning" into "message"
     And I click "submit-button"
-    And I see "System message set" in the page text
+    Then I see "System message set" in the page text
 
+  @focus
+    # due to home page being on different url from admin, this scenario has to be seperate, but relies on previous scenario having run
+  Scenario: System message should now be set on user-facing site. 
+    When I visit "/home"
+    Then I see "Your pizza is burning" in the page text
+    When I visit "/login"
+    Then I see "Your pizza is burning" in the page text
+    When I visit "/signup"
+    Then I see "Your pizza is burning" in the page text
+
+  @focus
+  Scenario: Remove second system message 
+    Given I visit the admin sign-in page
+    And I log in to admin
+    And I click "system-message-link"
     # remove message to display on front end
-    Then I clear the value in "message"
+    When I clear the value in "message"
     And I click "submit-button"
-    And I see "System message removed" in the page text
+    Then I see "System message removed" in the page text
 
+  @focus
+    # due to home page being on different url from admin, this scenario has to be seperate, but relies on previous scenario having run
+  Scenario: System message should now be removed on user-facing site. 
+    When I visit "/home"
+    Then I do not see "Your pizza is burning" in the page text
```

Comment:
what flavour? 🍕

Reply:
Ha ha, well actually this is either @Raffers or @townxelliot who came up with the idea of pizzas burning here originally :)